src/widgets/Tabs/PatientEdit.js

```diff
@@ -95,24 +97,20 @@ const PatientEditComponent = ({
               >
                 <View />
               </JSONForm>
-            </View>
-          )}
-        </View>
-
-        <FlexRow justifyContent="flex-end" alignItems="flex-end">
-          <PageButtonWithOnePress
-            text={buttonStrings.cancel}
-            onPress={onCancelPrescription}
-            style={{ marginRight: 7 }}
-          />
-          <PageButton
-            isDisabled={!canSaveForm}
-            text={buttonStrings.next}
-            onPress={savePatient}
-            style={{ marginLeft: 5 }}
-          />
-        </FlexRow>
-      </AfterInteractions>
+            </Animatable.View>
+          </AfterInteractions>
```

Comment:
Changed this to have the papers while the interaction was happening which then get 'filled' once the navigation interaction has completed - this makes the whole navigation interaction more fluid (opinion)